benchexec/tablegenerator/react-table/src/tests/__snapshots__/Summary.test.js.snap

```diff
@@ -3424,8 +3425,9 @@ exports[`Render Summary for big-table.diff.html 1`] = `
        >
         <div
          className="cell"
-         dangerouslySetInnerHTML={0}
-        />
+        >
+         -
```

Comment:
@PhilippWendler Theses were generated in the status columns. The python code still displays a "0" instead of a dash when it doesn't have any values in the status columns. I believe this is a bug? Are they handled separately in the python code?

Reply:
No, this should stay as it is. The status column *counts* the rows for the respective category, and the count over an empty list is of course 0. The other columns *sum* the values in the rows of the respective category, and there we can decide do have no result for a sum of an empty list to distinguish from the case where the sum happens to be 0. For the status column there are no such two cases that could be distinguished.

Reply:
Makes sense, i will adjust the code accordingly.